### ISSUE TITLE
cockpit error - 389-ds module

### ISSUE THREAD

**Issue by baffledmystic** [389ds/389-ds-base#5552]
rpm -qa | grep cockpit-389
cockpit-389-ds-1.4.3.31-6.module+el8dsrv+16980+c4b9cd33.noarch

Console logs show error in syntax for dsconf command:
```
bootstrap:1 CMD: loadReplSuffix: Load the replication info ==> dsconf -j ldapi://%2fvar%2frun%2fdsinsance.socket replication get-changelog --suffix o=archive
cockpit/$7f50cb6409a<E2><80><A6>sole/index.html#/:1 
       Uncaught SyntaxError: Unexpected token 'u', "usage: dsc"... is not valid JSON
    at JSON.parse (<anonymous>)
    at Function.<anonymous> (bootstrap:1:1)
    at cockpit.js:1:12867
    at cockpit.js:1:12978
    at L (cockpit.js:1:12383)
```

**Comment by droideck** [comment]
I can't reproduce the issue.
Could you please provide more info:

1. What other versions and packages do you have? 'rpm -qa | grep cockpit-389'
2. What are the reproduction steps exactly?

Also, if possible, please, try the new version - 389-ds-base-1.4.3.32-1.

**Comment by mreynolds389** [comment]
This was fixed in 1.4.3.31-8 (-9 is coming out soon), this build should be available soon...
